packages/react-components/react-switch/src/stories/Switch.stories.tsx

```diff
@@ -9,7 +9,6 @@ export { Disabled } from './SwitchDisabled.stories';
 export { Label } from './SwitchLabel.stories';
 export { LabelWrapping } from './SwitchLabelWrapping.stories';
 export { Required } from './SwitchRequired.stories';
-export { Themed } from './SwitchThemed.stories';
```

Comment:
This story is redundant and does not match patterns that we have for other components. I removed it to avoid confusion. To check how a component looks in different themes customers can use the theme selector and exists on every page:

![image](https://user-images.githubusercontent.com/14183168/170232327-46cacff3-cd93-49eb-849d-3c74d6bd4206.png)